### DISCUSSION TITLE
Welcome to Casper DAO Middleware Discussions!

### DISCUSSION THREAD

**Discussion by mrkara** [make-software/casper-dao-middleware#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by mikekeke** [comment]
Hey!
I was looking for some examples of DAO contracts on Casper network and bumped into your repositories.
I'm I understanding correctly, that this repository is something related to [dao-contracts](https://github.com/make-software/dao-contracts)? Some peace of logic that works with events, that contracts from `dao-contracts` emit?
(I'm just curious to learn more about everything related to Casper smart-contracts and DAO)

**Reply by mrkara** [reply]
Hey @mikekeke !

Thank you for stopping by.

Right. This repo is for the middleware which works with the smart contracts on the dao-contracts repository. The middleware makes it much easier for dApp developers to interact with the dao contracts through a friendly API.

**Reply by mikekeke** [reply]
Got it. Thanks!
Is it already deployed to the network to check it out, or this is work in progress?

**Reply by mrkara** [reply]
It's on TestNet at the moment. We are planning to deploy it on MainNet after extensive QA.

**Reply by mikekeke** [reply]
Does ` dao-contracts` is what https://portal.devxdao.com/ uses under the hood? Or is it general framework that will allow to add DAO capabilities to dApps?
